ops/demo/_config.tf

```diff
@@ -8,7 +8,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~>2.0"
+      version = "~>2.84.0"
```

Comment:
I think I may have misspoke when we talked about the versioning. Let's use `~> 2.84` instead of `2.84.0`, otherwise the minor version will never change - pessimistic constraint versioning only changes the rightmost value.

Reply:
We should do this everywhere we use the `~>` operator.

Reply:
Got it! I think that it's even better to let the minor version upgrade.